source/JsMaterialX/test/basics.spec.js

```diff
@@ -111,10 +111,16 @@ describe('Basics', () => {
 
     it('Matrices', () => {
         // Translation and scale
-        const trans = new mx.Matrix44().createTranslation(new mx.Vector3(1, 2, 3));
-        const scale = new mx.Matrix44().createScale(new mx.Vector3(2, 2, 2));
-        expect(trans.equals(new mx.Matrix44(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 1, 2, 3, 1)));
-        expect(scale.equals(new mx.Matrix44(2, 0, 0, 0, 0, 2, 0, 0, 0, 0, 2, 0, 0, 0, 0, 1)));
+        const trans = mx.Matrix44.createTranslation(new mx.Vector3(1, 2, 3));
```

Comment:
The tests seem to be a bit inconsistent. We don't use `new` on the matrices, but on the vectors. Furthermore, we use mix of `equals`, `equals.[].to.be.true` and `isEquivalent`.

Reply:
Oh, just noticed that `equals` and `isEquivalent` are mx methods, not mocha constructs...

Reply:
What is not consistent in regarding `new`? When creating Vectors AND Matrices directly via values we use the constructor. If we want to get special matrices like translations we use the specific creator functions

Reply:
Yep, my bad. I only saw the `new` being removed in front of the matrices, but overlooked that the methods have been changed into static functions in the bindings.